Sources/Stores/NetworkPersistableStore.swift

```diff
@@ -11,8 +11,8 @@ public enum NetworkPersistableStoreError: Swift.Error {
 }
 
 public class NetworkPersistableStore<Network: NetworkStack, Persistence: PersistenceStack>: NetworkStore
-where Network.Remote == Data, Network.Request == URLRequest, Network.Response == URLResponse,
-      Persistence.Remote == Data {
+    where Network.Remote == Data, Network.Request == URLRequest, Network.Response == URLResponse,
```

Comment:
here we are starting to see the weird line breaking cases (and possibilities) that come with generics and type declarations 😅 

Personally I'm a fan of aligning the `where` at the same level as `func`, but once again this is not defined anywhere on our style guide AFAIK.

Reply:
I am reverting these changes (the ones I find at least). 
I don't disagree with you. Xcode's built in "formatter" was the one you indented the `where` clause.

Reply:
yeah, Xcode can be quite "opinionated" on indentation 😝